### ISSUE TITLE
[Bug] Multiple left-hand assignment require case, throws error.

### ISSUE THREAD

**Issue by adam-cyclones** [wessberg/cjs-to-esm-transformer#6]
I am building a bundle-less bundler using your wonderful project (thank you!), I have run into an issue with a case that looks like this:  
``` js
const constants = exports.constants = require('./constants.js')
```
### What is the problem
- This is an extract from a CJS module (shall we say ... loose code style), this library is called minizlib.
- The snippet is valid JS although less common, I'd love to help you with a PR but not sure where to start?
- Throwing a stack trace. I don't know but my best guess is that this case is not handled?

```
TypeError: Cannot read property 'kind' of undefined
    at Object.isExpressionNode (/Users/me/Code/typescript/unpack/node_modules/typescript/lib/typescript.js:14615:33)
    at isExpression (/Users/me/Code/typescript/unpack/node_modules/@wessberg/cjs-to-esm-transformer/src/transformer/util/is-expression.ts:8:81)
    at visitBinaryExpression (/Users/me/Code/typescript/unpack/node_modules/@wessberg/cjs-to-esm-transformer/src/transformer/before/visitor/visit/visit-binary-expression.ts:295:76)
    at visitNode (/Users/me/Code/typescript/unpack/node_modules/@wessberg/cjs-to-esm-transformer/src/transformer/before/visitor/visit/visit-node.ts:24:10)
    at /Users/me/Code/typescript/unpack/node_modules/@wessberg/cjs-to-esm-transformer/src/transformer/before/transform-source-file.ts:212:26
    at visitNode (/Users/me/Code/typescript/unpack/node_modules/typescript/lib/typescript.js:74775:23)
    at Object.visitEachChild (/Users/me/Code/typescript/unpack/node_modules/typescript/lib/typescript.js:75140:224)
    at childContinuation (/Users/me/Code/typescript/unpack/node_modules/@wessberg/cjs-to-esm-transformer/src/transformer/before/transform-source-file.ts:209:15)
    at visitVariableDeclaration (/Users/me/Code/typescript/unpack/node_modules/@wessberg/cjs-to-esm-transformer/src/transformer/before/visitor/visit/visit-variable-declaration.ts:29:10)
    at visitNode (/Users/me/Code/typescript/unpack/node_modules/@wessberg/cjs-to-esm-transformer/src/transformer/before/visitor/visit/visit-node.ts:22:10)
```

Id probably want to see an output fixed up like this:

``` js
export constants from './constants.js';
```

or even

``` js
import constants from './constants.js';
export constants;
```

What do you think, hope you are still active.

**Comment by adam-cyclones** [comment]
l will probably fork and try and figure this out.

**Comment by wessberg** [comment]
Hey there. Thanks for submitting this issue! No need to close it, I'm looking into the issue and will release a new version once it has been fixed.

**Comment by wessberg** [comment]
This has been implemented and released as part of [v0.0.21](https://github.com/wessberg/cjs-to-esm-transformer/releases/tag/v0.0.21).

Specifically, I've added support for the general case of `exports.<binding> = require("...")`, including `const <a> = exports.<b> =  require("...")`

Now, I'm not sure if the error you shared will persist after this version, but please let me know if you have any additional problems.

**Comment by adam-cyclones** [comment]
> This has been implemented and released as part of [v0.0.21](https://github.com/wessberg/cjs-to-esm-transformer/releases/tag/v0.0.21).
> 
> Specifically, I've added support for the general case of `exports.<binding> = require("...")`, including `const <a> = exports.<b> = require("...")`
> 
> Now, I'm not sure if the error you shared will persist after this version, but please let me know if you have any additional problems.

Thank you so much! I really am happy with this outcome. I didn't fork in the end but always happy to find more issues. I am working on didi (a module un-bundler for the web) which is using your work here so its wonderful that I can get rid of this edge case.
